cmd/collectors.go

```diff
@@ -64,7 +64,9 @@ func parseCollector(s string) (t, arg string) {
 	}
 }
 
-func newCollector(collectorName, arg string, src *loader.SourceData, conf Config) (lib.Collector, error) {
+func newCollector(
```

Comment:
I've intentionally not silenced this warning, since we should definitely fix the code, but I don't plan on doing it in this PR, so I'll resolve the comment. Not sure if that would make it go away or if I'd have to silence it and add a todo...